src/reports/components/report-sections/make-details-section-fc.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import { CommentIcon } from 'common/icons/comment-icon';
+import { DateIcon } from 'common/icons/date-icon';
+import { UrlIcon } from 'common/icons/url-icon';
+import { NamedFC, ReactFCWithDisplayName } from 'common/react/named-fc';
+import { ScanMetaData } from 'common/types/store-data/scan-meta-data';
+import { isEmpty } from 'lodash';
+import * as React from 'react';
+import { SectionProps } from './report-section-factory';
+
+export type DetailsSectionProps = Pick<
+    SectionProps,
+    'scanMetadata' | 'description' | 'scanDate' | 'toUtcString'
+>;
+
+export type ScanDetailInfo = {
+    label: string;
+    content: string | JSX.Element;
+};
+
+export function makeDetailsSectionFC(
+    getDisplayedScanTargetInfo: (scanMetaData: ScanMetaData) => ScanDetailInfo,
```

Comment:
You're using this type in a few places; might not be a bad idea to name it and have it separately.